src/Client.php

```diff
@@ -264,16 +271,14 @@ private function addThrowableToEvent(Event $event, \Throwable $exception): void
                 'value' => $this->serializer->serialize($currentException->getMessage()),
             ];
 
-            if ($this->getOptions()->getAutoLogStacks()) {
-                $data['stacktrace'] = Stacktrace::createFromBacktrace(
-                    $this->getOptions(),
-                    $this->serializer,
-                    $this->representationSerializer,
-                    $currentException->getTrace(),
-                    $currentException->getFile(),
-                    $currentException->getLine()
-                );
-            }
+            $data['stacktrace'] = Stacktrace::createFromBacktrace(
```

Comment:
Why did we loose the `if` check here?

Reply:
We don't need this check, if there is an exception we always want the stacktrace.

Reply:
`attachStacktrace` only determines if a stacktrace is attached to `captureMessage`